Cargo.toml

```diff
@@ -409,3 +409,8 @@ xcm-procedural = { git = "https://github.com/subspace/polkadot-sdk", rev = "949b
 # De-duplicate extra copy that comes from Substrate repo
 [patch."https://github.com/subspace/polkadot-sdk.git"]
 substrate-bip39 = "0.6.0"
+
+[patch.crates-io]
+# Reason: We need to patch subxt dependency of substrate to use our fork
+# TODO: Remove if/when we are using upstream substrate instead of fork
+sp-crypto-hashing = { git = "https://github.com/subspace/polkadot-sdk", rev = "949bb3cfb64ab974e4fc49328fb3e81c96bc0fbe" }
```

Comment:
I don't think we'll use upstream Substrate any time soon, but even if we do, I'm not sure they actually publish all crates to crates.io anyway. This dependency is not going away. Also since versions are only bumped periodically, there is a small chance of incompatibility. And in the end it is a single small dependency, I'm not sure it is worth messing with for such a tiny "win".